backend/core/types/src/backend-swagger.ts

```diff
@@ -848,6 +848,29 @@ export class UserService {
 
       let data = params.body
 
+      configs.data = data
```

Comment:
Do you think the diff is so large in this file just because things were moved around rather than changed?

Reply:
@seanmalbert I think you will answer this question better than me :)

Reply:
Yes, for some reason whenever I run generate client on this branch, it creates a whole mess with duplicate entries that have to be resolved. Even after a rebase. It's really the only reason why I'm hesitant to merge this in. @emilyjablonski , can you run `yarn generate:client` on this and see if you see the same thing?

Reply:
I do get duplicates locally on this branch when I generate the client

Reply:
I just talked to @pbn4 about this, and I now have clarity on why this is happening. Models, instead of DTOs are getting exposed to swagger in a couple of cases, which causes "duplicate" entries, one from the model and one from the dto, so I'll patch that up.